packages/app/src/cli/prompts/dev.test.ts

```diff
@@ -86,6 +125,27 @@ describe('selectOrganization', () => {
   })
 })
 
+describe('selectFunctionRun', () => {
```

Comment:
I noticed the name of this file is `dev.test.ts`- is our prompt defined in the right place?

Reply:
It is among the other prompts so I think so